SECURITY.md

```diff
@@ -0,0 +1,59 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+and:
+    - Navigate to the security tab on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `security@crossplane.io` detailing the issue and steps
+to reproduce.
+
+The reporter(s) can expect a response within 24 hours acknowledging
+the issue was received. If a response is not received within 24 hours, please
+reach out to any committer directly to confirm receipt of the issue.
+
+## Review Process
+
+Once a committer has confirmed the relevance of the report, a draft security
```

Comment:
Perhaps maintainer rather than committer?

Reply:
+1, can we update each usage of 'committer" to be "maintainer" instead? that's the formal position we recognize in our governance.